src/app/tests/suites/certification/Test_TC_TCCM_3_1.yaml

```diff
@@ -21,183 +20,111 @@ PICS:
-
-          [1690190348.956716][16435:16437] CHIP:DMG: }
-          [1690190348.956853][16435:16437] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0052 Attribute 0x0000_0000 DataVersion: 2612443001
-          [1690190348.956895][16435:16437] CHIP:TOO:   SupportedModes: 3 entries
-          [1690190348.956919][16435:16437] CHIP:TOO:     [1]: {
-          [1690190348.956924][16435:16437] CHIP:TOO:       Label: Normal
-          [1690190348.956929][16435:16437] CHIP:TOO:       Mode: 0
-          [1690190348.956935][16435:16437] CHIP:TOO:       ModeTags: 1 entries
-          [1690190348.956952][16435:16437] CHIP:TOO:         [1]: {
-          [1690190348.956955][16435:16437] CHIP:TOO:           Value: 0
-          [1690190348.956958][16435:16437] CHIP:TOO:          }
-          [1690190348.956961][16435:16437] CHIP:TOO:      }
-          [1690190348.956967][16435:16437] CHIP:TOO:     [2]: {
-          [1690190348.956970][16435:16437] CHIP:TOO:       Label: Rapid Cool
-          [1690190348.956972][16435:16437] CHIP:TOO:       Mode: 1
-          [1690190348.956976][16435:16437] CHIP:TOO:       ModeTags: 1 entries
-          [1690190348.956980][16435:16437] CHIP:TOO:         [1]: {
-          [1690190348.956982][16435:16437] CHIP:TOO:           Value: 16384
-          [1690190348.956984][16435:16437] CHIP:TOO:          }
-          [1690190348.956987][16435:16437] CHIP:TOO:      }
-          [1690190348.956992][16435:16437] CHIP:TOO:     [3]: {
-          [1690190348.956994][16435:16437] CHIP:TOO:       Label: Rapid Freeze
-          [1690190348.956997][16435:16437] CHIP:TOO:       Mode: 2
-          [1690190348.957000][16435:16437] CHIP:TOO:       ModeTags: 1 entries
-          [1690190348.957003][16435:16437] CHIP:TOO:         [1]: {
-          [1690190348.957006][16435:16437] CHIP:TOO:           Value: 16385
-          [1690190348.957008][16435:16437] CHIP:TOO:          }
-          [1690190348.957010][16435:16437] CHIP:TOO:      }
-      disabled: true
+      command: "readAttribute"
+      attribute: "SupportedModes"
+      response:
+          constraints:
+              type: list
+              minLength: 2
 
     - label:
-          "Step 5: TH sends a ChangeToMode command to the DUT with NewMode set
-          to new_mode_th"
+          "Select a value from supported_modes_dut different from on_mode_dut.
+          Save the value as new_mode_th. TH sends a ChangeToMode command to the
+          DUT with NewMode set to new_mode_th"
+      runIf: IsExpectedValue
       PICS: TCCM.S.C00.Rsp && TCCM.S.F00
-      verification: |
-          ./chip-tool refrigeratorandtemperaturecontrolledcabinetmode change-to-mode 2 1 1
-
-          Verify on TH(chip-tool) log, DUT responds contains a ChangeToModeResponse command with a SUCCESS (value 0x00) status response and below is the sample log provided for the raspi platform:
+      command: "ChangeToMode"
+      arguments:
+          values:
+              - name: "NewMode"
+                value: new_mode_th
```

Comment:
Test plan says to check the response fields, but this is not checking them.  Why not?

Reply:
Updated.